src/phoenix/datasets/experiments.py

```diff
@@ -321,21 +339,29 @@ def evaluate_experiment(
             .get("examples", [])
         )
     ]
-    _evaluate_experiment(experiment, evaluators, dataset_examples, client)
-
-
-ExperimentEvaluatorName: TypeAlias = str
+    _evaluate_experiment(
+        experiment,
+        evaluators=evaluators,
+        dataset_examples=dataset_examples,
+        client=client,
+    )
 
 
 def _evaluate_experiment(
     experiment: Experiment,
-    evaluators: Union[ExperimentEvaluator, Iterable[ExperimentEvaluator]],
+    *,
+    evaluators: Optional[Evaluators] = None,
+    llm_evaluators: Optional[Evaluators] = None,
```

Comment:
I find it odd that we are passing in the `llm_evaluators` separately from the other evaluators. Are we not able to infer the annotator kind from the `kind` property on the base class?

Reply:
We can't do that if it's just a raw function. The user would have to let us know.

Reply:
I see. I think it would be more straightforward to just require users to decorate their functions with `create_evaluator` then.

Reply:
if they have multiple, it would be more of a hassle (and more code) to do it that way

Reply:
If I understand, the main reason for having different arguments here is to be able to known the annotator kind when the user passes a function instead of an evaluator class. But it also makes it possible for the user to pass code evaluators as having LLM kind and vice-versa, doesn't it?

Reply:
Yes, It's optional, but it's a way for users to express their intentions if they so choose.

Reply:
I think we can probably just keep the `evaluators` argument and have raw functions default to `CODE`. If the user wants to define an LLM evaluator with a function, they can do so via the decorator. Keeps the signature of `run_experiment` and `evaluate_experiment` simpler.

Reply:
I think that's a bit too tightfisted. It's not really make making the signature that much more complicated, and it's one less thing (i.e. decorators and whatnot) for the users to think about. The users have to find and import the right decorator and then apply hem appropriately. What if we change the decorators? It's all just unnecessary friction.